gr-fft/python/fft/__init__.py

```diff
@@ -22,3 +22,28 @@
     dirname, filename = os.path.split(os.path.abspath(__file__))
     __path__.append(os.path.join(dirname, "..", "..", "swig"))
     from .fft_swig import *
+
+# Shim the appropriate python fft library here
+fft_pylib = None
+try:
+    import numpy as np
+except ImportError:
+    exit    
+
+try:
+    from numpy.fft import fftpack as fft_pylib
+except ImportError:
+    # Numpy changed fft implementation in version 1.17
+    # from fftpack to pocketfft
+    try:
+        from numpy.fft import pocketfft as fft_pylib
+    except ImportError:
+        try:
+            from numpy.fft import fft as fft_pylib
+        except ImportError:
+            try:
+                from scipy import fft as fft_pylib
+            except ImportError:
+                pass
+                # out of options, fail silently 
```

Comment:
Should we fail silently? It seems we're asking our tools to rely on this. Plus, numpy is a dependency of GNU Radio and we'd fail elsewhere if it weren't there.